doc/manual.rst

```diff
@@ -4893,6 +4893,8 @@ no semantics outside of a template definition and cannot be abstracted over:
 To get rid of hygiene in templates, one can use the `dirty`:idx: pragma for
 a template. ``inject`` and ``gensym`` have no effect in ``dirty`` templates.
 
+``gensym``'ed symbols cannot be used
```

Comment:
```suggestion
``gensym`` processed symbols cannot be used.
```

Yo 'cause slang kinda hard 2 read 4 n00bz. 🤪

Reply:
It's work-in-progress to remind me to write this part of the spec.